setup/install_che/codewind-clusterrole.yaml

```diff
@@ -67,3 +67,15 @@ rules:
 - apiGroups: ["route.openshift.io"]
   resources: ["routes", "routes/custom-host"]
   verbs: ["get", "list", "create", "delete", "watch", "patch", "update"]
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: codewind-rolebinding
+subjects:
+- kind: ServiceAccount
+  name: codewind
```

Comment:
Should we name it `eclipse-codewind` for consistency with the cluster role?

Reply:
Renamed.